jobs/rocoto/coupled_ic.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
+
+set -x
+
+###############################################################
+## Abstract:
+## Create FV3 initial conditions from GFS intitial conditions
+## RUN_ENVIR : runtime environment (emc | nco)
+## HOMEgfs   : /full/path/to/workflow
+## EXPDIR : /full/path/to/config/files
+## CDATE  : current date (YYYYMMDDHH)
+## CDUMP  : cycle name (gdas / gfs)
+## PDY    : current date (YYYYMMDD)
+## cyc    : current cycle (HH)
+###############################################################
+
+###############################################################
+# Source FV3GFS workflow modules
+. $HOMEgfs/ush/load_fv3gfs_modules.sh
+status=$?
+[[ $status -ne 0 ]] && exit $status
+err=0
+
+export DATAROOT="$RUNDIR/$CDATE/$CDUMP"
```

Comment:
Do you need `export` here?

Reply:
Actually, we don't need this at all. The directory is created and then never used.